docs/installation/debian.md

```diff
@@ -75,6 +75,11 @@ from the new repository:
 
             deb https://apt.dockerproject.org/repo debian-stretch main
 
+> **Note**: Docker does not provide packages for all architectures. To install docker on
+> a multi-architecture system, add an `[arch=...]` clause to the entry. Refer to the
+> [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
+> for details.
+
```

Comment:
Unfortunately, it has to be 4 spaces in Markdown, otherwise it still breaks the numbered list; see the screenshot;

<img width="596" alt="screen shot 2015-11-25 at 01 45 12" src="https://cloud.githubusercontent.com/assets/1804568/11385157/563ea6de-9316-11e5-8d19-e66e0f0f73ce.png">